core/src/main/scala-2.13+/cats/instances/package.scala

```diff
@@ -0,0 +1,62 @@
+package cats
+
+package object instances {
```

Comment:
Can we use an abstract class for objects from both scala specific versions to inherit from and remove the duplication?

Reply:
Not without dropping in some `lazy val`s. Here's a minimization:
```scala
package foo

package object bar extends BarPackage
package bar {
  abstract class BarPackage {
    object baz extends Baz
  }
  trait Baz {
    val x = foo.bar.baz
  }
}
```
Referring to `baz` will stack overflow.